transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -350,14 +350,9 @@ private void closeAll() {
         } catch (IOException ignore) {
             // ignore on close
         }
-        Collection<AbstractEpollChannel> array = new ArrayList<AbstractEpollChannel>(channels.size());
-
-        for (AbstractEpollChannel channel: channels.values()) {
-            array.add(channel);
-        }
-
-        for (AbstractEpollChannel ch: array) {
-            ch.unsafe().close(ch.unsafe().voidPromise());
+        for (PrimitiveEntry<AbstractEpollChannel> entry : channels.entries()) {
```

Comment:
You are right, guys. Thanks for not letting me do something stupid!
I updated a PR.